bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+import numpy as np
+import h5py
+import operator
+import argparse
+import logging
+from awkde import GaussianKDE
```

Comment:
It looks like the pycbc install build is failing (see the tests on the webpage for this PR) because the current automatically built environment does not include the awkde package.  Can you put in another import_optional for awkde.GaussianKDE rather than do this import?
Also can you add sklearn and awkde packages to the file 'companion.txt' in the main pycbc code directory?   I think this tells the automatic build system that they may be optionally imported by some codes.